### PR TITLE
Add custom noscript page for Javascript disabled browsers

### PR DESCRIPTION
**Pull request by hasanakg**
Bug: T343344

### CODE REVIEW

File: resources/sass/noscript.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import '~@wikimedia/codex/dist/codex.style.css';
```
Comment (chukarave) [inline resources/sass/noscript.scss:1]: Do you know why we have to import from `/dist/`? I saw we do it for the app css too but it's not how the [codex docs](https://doc.wikimedia.org/codex/main/using-codex/usage.html#using-design-tokens) advise to do it

File: resources/views/app.blade.php [hunk 2]
```diff
@@ -5,6 +5,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
     <link href="{{ mix('/css/app.css') }}" rel="stylesheet" />
     <script src="{{ mix('/js/app.js') }}" defer></script>
+
+    <noscript>
```
Comment (chukarave) [inline resources/views/app.blade.php:9]: My personal preference would be to include this in a separate template that gets included here. less clutter

**Review by chukarave — COMMENTED** [review]
It works well for me locally, as far as I can tell the styles in the breakpoints are correct. some very minor things.

File: resources/views/app.blade.php (hunk 2 above)
Comment (hasanakg) [reply resources/views/app.blade.php:9]: You mean having a separate `noscript.blade.php` file right? We already have a separate `scss` file for that, so it sounds better for me as well. If that's the case, just let me know so I can update the code.

Comment (chukarave) [reply resources/views/app.blade.php:9]: yes that's exactly what i mean!

**Review by chukarave — APPROVED** [review]
The latest version looks good to me.

**Pull request merged**